README.md

```diff
@@ -84,9 +82,100 @@ But you'll still need to pick the built version (which ships with the package).
 
 ```javascript
 // with the classic require...
-let Kuzzle = require('kuzzle-sdk/dist/kuzzle.js')
+const { Kuzzle } = require('kuzzle-sdk/dist/kuzzle.js')
 // ... or with the new import directive.
-import Kuzzle from 'kuzzle-sdk/dist/kuzzle.js'
+import { Kuzzle } from 'kuzzle-sdk/dist/kuzzle.js'
+```
+
+## Protocols used
+Actuellement, le SDK supporte nativement 3 protocols: `http`, `websocket` et `socketio`.  
+
+The main reason behind this is that while Socket.IO offers better compatibility with older web browsers, our raw WebSocket implementation is about 20% faster
```

Comment:
nitpicking: remove `The main reasion behind this is that`

just be decalarative, something like that:
```
Websocket and Socket.IO protocols implement the whole Kuzzle API, while HTTP protocol does not implement realtime features (rooms and subscriptions)
While Socket.IO offers better compatibility with older web browsers, our raw WebSocket implementation is about 20% faster
```